apps/console/src/features/identity-providers/components/wizards/apple/apple-authentication-provider-create-wizard-content.tsx

```diff
@@ -63,6 +63,10 @@ interface AppleAuthenticationProviderCreateWizardContentPropsInterface extends I
      * @param values - Form values.
      */
     onSubmit: (values: AppleAuthenticationProviderCreateWizardFormValuesInterface) => void;
+    /**
+     * On focus callback.
+     */
+    setOnFocus: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
AFAIU, this function is to keep track of the current step in the guide in the create IDP side panel. In that case, shall we rename this prop name to be more descriptive, eg: `setCurrentStepInGuidePanel` etc?